examples/axum-app/src/main.rs

```diff
@@ -0,0 +1,173 @@
+use axum::extract::{Path, Query};
+use axum::http::StatusCode;
+use axum::response::{Html, IntoResponse, Redirect, Response};
+use axum::routing::get;
+use axum::{Router, serve};
+use rinja::Template;
+use serde::Deserialize;
+use tower_http::trace::TraceLayer;
+use tracing::{Level, info};
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    tracing_subscriber::fmt()
+        .with_max_level(Level::DEBUG)
+        .init();
+
+    let app = Router::new()
+        .route("/", get(start_handler))
+        .route("/{lang}/index.html", get(index_handler))
+        .route("/{lang}/greet-me.html", get(greeting_handler))
+        .fallback(|| async { AppError::NotFound })
+        .layer(TraceLayer::new_for_http());
+
+    // In a real application you would most likely read the configuration from a config file.
+    let listener = tokio::net::TcpListener::bind("127.0.0.1:8000")
+        .await
+        .map_err(Error::Bind)?;
+
+    if let Ok(addr) = listener.local_addr() {
+        info!("Listening on http://{addr}/");
+    }
+    serve(listener, app).await.map_err(Error::Run)
+}
+
+#[derive(displaydoc::Display, pretty_error_debug::Debug, thiserror::Error)]
+enum Error {
+    /// could not bind socket
+    Bind(#[source] std::io::Error),
+    /// could not run server
+    Run(#[source] std::io::Error),
+}
+
+/// Using this type your user can select the display language of your page.
```

Comment:
```suggestion
/// Thanks to this type, your user can select the display language of your page.
```

Sentence sounded a bit weird.